script.js

```diff
@@ -0,0 +1,78 @@
+function createCourse(course) {
+    let html = 
+        `
+            <div class="course">
+                <a href="#">
+                    <img src="${course?.image??"assets/PLMP"}" alt="courseImg" width="100%">
+                    <div class="course-text">
+                        <h3>${course?.title??"Course Name"}</h3><span class="instructorName">${course?.instructor??"Jeff Bezos"}</span>
+                        <div>
+                            <span class="rating">${course?.rating??4.5}</span>
+                            ${course.rating>=1?`<span class="fa fa-star checked"></span>`:`<span class="fa fa-star"></span>`}
+                            ${course.rating>=2?`<span class="fa fa-star checked"></span>`:`<span class="fa fa-star"></span>`}
+                            ${course.rating>=3?`<span class="fa fa-star checked"></span>`:`<span class="fa fa-star"></span>`}
+                            ${course.rating>=4?`<span class="fa fa-star checked"></span>`:`<span class="fa fa-star"></span>`}
+                            ${course.rating>=5?`<span class="fa fa-star checked"></span>`:`<span class="fa fa-star"></span>`}
+                        </div>
+                        <h3>${course?.price??"Free"}</h3>
+                    </div>
+                </a>
+            </div>
+            `;
+    return html;
+}
+let currentTab = "Python";
+function updateCourses(courses){
+    
+    let coursesContainer = document.querySelector(".courses-container");
+    coursesContainer.innerHTML = "";
```

Comment:
You can remove phase#1 courses from `index.html` to avoid the visual glitch of having them then a split second later new courses with different thumbnails load.

Reply:
Saw it in phase 3 ✅